src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -925,16 +934,17 @@ public String getCallSettingsTypeName(Interface apiInterface) {
   }
 
   /** The name of the return type of the given grpc streaming method. */
-  public String getGrpcStreamingApiReturnTypeName(Method method, ImportTypeTable typeTable) {
+  public String getGrpcStreamingApiReturnTypeName(
+      MethodContext methodContext, ImportTypeTable typeTable) {
```

Comment:
I haven't yet run into a case where the coupling is a problem. Once I do, I'd be happy to revert to using the minimal parameter scope then.